config/initializers/assets.rb

```diff
@@ -12,3 +12,4 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+# Rails.application.config.assets.precompile += %w( devise_user.scss )
```

Comment:
Is this how you precompile assets also in Heroku? lol, that's been giving me a hard time :P

Reply:
yeah, took a lot of googling to figure it out finally